examples/Cargo.toml

```diff
@@ -37,7 +37,7 @@ esp-hal-embassy     = { path = "../esp-hal-embassy", optional = true }
 esp-ieee802154      = { path = "../esp-ieee802154", optional = true }
 esp-println         = { path = "../esp-println", features = ["log"] }
 esp-storage         = { path = "../esp-storage", optional = true }
-esp-wifi            = { path = "../esp-wifi", optional = true }
+esp-wifi            = { path = "../esp-wifi", features = ["log"], optional = true }
```

Comment:
:sweat_smile: I spent far to long wondering why I wasn't getting log messages.

Reply:
This was an intentional change, though.

Reply:
> This was an intentional change, though.

It was intentional in the main crate, but we didn't update the examples accordingly

Reply:
Not reading carefully enough, again... sorry.